modules/appeals_api/app/controllers/appeals_api/v2/decision_reviews/contestable_issues_controller.rb

```diff
@@ -17,8 +17,6 @@ class ContestableIssuesController < AppealsApi::ApplicationController
       before_action :validate_json_schema, only: %i[index]
       before_action :validate_params, only: %i[index]
 
-      EXPECTED_HEADERS = %w[X-VA-SSN X-VA-Receipt-Date X-VA-File-Number].freeze
```

Comment:
Replaced with `HEADERS` which pulls headers from the schema (and returns the same array).